Fix usage of context in Django 1.10

izquierdo opened the pull request:
This upgrades the code to use the new template system introduced in Django 1.8, as support for the old system was removed in Django 1.10.

See https://docs.djangoproject.com/en/1.10/ref/templates/upgrading/#django-template-loader

shtalinberg commented:
Hello 
All pull requests should pass all tests with Travis-CI  
Now All checks have failed - it's bad

izquierdo commented:
Yep, sorry, noticed after submitting it. Working on a fix.
>
>

izquierdo commented:
Updated and fixed. Main changes are:

* Making `context` mandatory in `PageList`
* Using `context.flatten()` to pass a plain dictionary to `render`.
